pkg/kubectl/rollback.go

```diff
@@ -283,6 +286,80 @@ func (r *DaemonSetRollbacker) Rollback(obj runtime.Object, updatedAnnotations ma
 	return rollbackSuccess, nil
 }
 
+type StatefulSetRollbacker struct {
+	c clientset.Interface
+}
+
+func (r *StatefulSetRollbacker) Rollback(obj runtime.Object, updatedAnnotations map[string]string, toRevision int64, dryRun bool) (string, error) {
```

Comment:
A comment here might help, to indicate that `toRevision` is a non-negative integer, with 0 being reserved to indicate that it must be rolled back to previous configuration.